shacl12-sparql/index.html

```diff
@@ -1715,6 +1715,11 @@ <h2>Pre-binding of Variables in SPARQL Queries</h2>
 			<p>
 				Some features of SHACL-SPARQL rely on the concept of <a>pre-binding of variables</a> as defined in this section.
 			</p>
+			<p class="issue">
```

Comment:
I found an example of this; this can be added:

```suggestion
			<p class="issue atrisk" data-number="647">
```